src/AmbientSounds.Uwp/Views/ShellPage.xaml

```diff
@@ -193,6 +193,13 @@
                                 Duration="0:0:0.5" />
                         </animations:Implicit.ShowAnimations>
                     </controls:SaveMixButton>
+                    <Button
+                        AutomationProperties.Name="{x:Bind strings:Resources.PlayRandom}"
+                        Click="{x:Bind ViewModel.PlayRandomSoundAsync}"
+                        Style="{StaticResource SubtleIconButton}"
+                        ToolTipService.ToolTip="{x:Bind strings:Resources.PlayRandom}">
+                        <FontIcon FontFamily="{StaticResource FluentUIGlyphs}" Glyph="&#xE14B;" />
```

Comment:
can you go to app.xaml and add a glyph entry for `&#xE14B;`. If you go to app.xaml and search for glyph, you'll find a bunch of entries so you can see how it's used. 

Then in this line, you can just do `Glyph="{StaticResource <NameOfYourGlyph>}"`